cmd/gpu-feature-discovery/main.go

```diff
@@ -166,7 +170,15 @@ func start(c *cli.Context, cfg *Config) error {
 		}
 		klog.Infof("\nRunning with config:\n%v", string(configJSON))
 
-		manager := resource.NewManager(config)
+		// TODO: If applicable, an nvmllib needs to be passed to the infolib constructor.
```

Comment:
What do you mean by "if applicable"? When is this necessary vs. not?

Reply:
I think this comment is not relevant anymore. This was an earlier itteration.

Reply:
Removed the comment.